src/index.js

```diff
@@ -7,8 +7,8 @@ export {default as makeConfig} from './makeConfig'
 export {default as output} from './output'
 export {default as server} from './server'
 export {default as babelLoader} from './babelLoader'
-export {default as isomorphic} from './isomorphic'
-export {default as isomorphicConfig} from './isomorphic-config'
+//export {default as isomorphic} from './isomorphic'
+//export {default as isomorphicConfig} from './isomorphic-config'
```

Comment:
I'm not sure if these were actually taken advantage of in any project other than the generated manifest file in vacuumlabs-website to automatically get the generated hashed filenames. It would be nice to have a replacement in place at least for that.